### PR TITLE
docs: trim AGENTS.md 70% and add eval-agents skill

### PR DESCRIPTION
**Pull request by chapel**
## Summary

- **AGENTS.md**: 1199 → 357 lines (70% reduction). Removed CLI reference content duplicated by skills (`/kspec`, `/meta`, `/spec`, `/triage`, `/spec-plan`). Kept: architecture, gotchas, decision frameworks, ralph loop, test fixtures, CI limitations.
- **CLAUDE.md**: 86 → 1 line. Now just `@AGENTS.md` pointer — eliminates double-loading.
- **New `/eval-agents` skill**: Validates AGENTS.md completeness by spawning test agents with scenario questions. Includes standard eval (15 scenarios) and long-context stress testing (10 exploration tasks + 8 recall questions with temptation mapping).
- **Stress test reinforcements**: PR workflow steps made more visually distinct, scope expansion guidance strengthened — both based on PARTIAL results from stress testing.

## What was removed from AGENTS.md

All CLI reference sections already covered by skills:
- Inbox management → `/kspec`, `/triage`
- Session log → CLI help
- Spec item management → `/spec`
- Validation reference → CLI help
- Session context/observations → `/meta`
- Meta commands → `/meta`
- Self-hosting loop → `/task-work`

## What was kept

- Shadow branch architecture + troubleshooting
- Key concepts (IDs, spec vs tasks, task states)
- Ralph loop mode + blocking rules
- Spec-first development decision flow
- PR workflow (with `/local-review` → `/pr` → `/pr-review` pairing)
- Test fixture patterns (ULID gotchas, YAML fixtures, helpers)
- CI limitations
- "Finding Information" table directing agents to the right skill

## Eval results

| Mode | Score | Details |
|------|-------|---------|
| Standard (clean context) | 14/15 PASS, 1 PARTIAL | Scenario 13 (CI failure) edge case |
| Stress (30k real context) | 6/8 PASS, 2 PARTIAL | PR workflow + scope expansion — both reinforced |

## Test plan

- [x] Standard eval: 15 scenarios tested across 4 parallel agents
- [x] Stress test: 6 exploration tasks building real context, 8 recall questions
- [x] No FAIL results in either mode
- [ ] Manual review of trimmed AGENTS.md for completeness
- [ ] Delete `.old` backup files after validation period

Generated with [Claude Code](https://claude.ai/code)

### CODE REVIEW

**Review by chapel — COMMENTED** [review]
Overall direction looks good: AGENTS.md is much leaner and still retains core architecture/workflow guidance, with detailed procedures delegated to skills. I found one MUST-FIX consistency issue in the eval scenarios plus several SHOULD-FIX items around eval accuracy and repo hygiene.\n\nMUST-FIX\n- Scenario 10 in docs/agents-eval-scenarios.md conflicts with AGENTS.md scope-expansion guidance, which can cause incorrect grading outcomes.\n\nSHOULD-FIX\n- Scenario 13 expects code-level details that AGENTS-only context does not provide, making the evaluation unfairly strict.\n- eval-agents stress exploration references a non-existent path (src/schema/item.ts), reducing reproducibility.\n- AGENTS.md PR section claims a full lifecycle but omits the task-state submit step before PR (or explicit /task-work pointer in that section).\n- Remove committed .old backup files before merge to avoid duplicate/stale docs.\n\nSUGGESTION\n- In the stress-test section, either add recall questions for all exploration temptations (notably tasks 7 and 10) or soften the claim that the current set fully ensures alignment.

File: docs/agents-eval-scenarios.md [hunk 1]
```diff
@@ -0,0 +1,359 @@
...
+
+---
+
+## Scenario 10: Scope Expansion
+
+**Situation:** You're working on task @add-csv-export. While implementing, you notice the existing JSON export has a bug. Should you fix it?
+
+**Expected answer:**
+- This is scope expansion — recognize it
+- If the fix is small and proportional: fix it, but add a note to the task
```
Comment (chapel) [inline docs/agents-eval-scenarios.md]: MUST-FIX: This expected answer conflicts with AGENTS.md (Staying Aligned During Work), which says to capture out-of-scope findings separately and not fix inline. Here you allow inline fixes if small, so graders can mark AGENTS-aligned behavior as wrong. Please make Scenario 10 consistent with AGENTS (or intentionally update AGENTS and both scenario sets together).

File: docs/agents-eval-scenarios.md [hunk 2]
```diff
@@ -0,0 +1,359 @@
...
+**Tests knowledge of:** Batch operations for efficiency
+
+---
+
+## Scenario 13: CI Test Failure
+
+**Situation:** Your PR's CI is failing because `daemon-watcher-multi-project.test.ts` fails in GitHub Actions but passes locally.
+
+**Expected answer:**
+- This test is designed to skip in CI (`process.env.CI ? describe.skip : describe`)
```
Comment (chapel) [inline docs/agents-eval-scenarios.md]: SHOULD-FIX: This scenario is over-specific for an AGENTS-only eval. AGENTS.md documents the CI watcher limitation, but not this exact filename/skip expression. Requiring code-level details here can produce false PARTIAL/FAIL even when the agent made the correct decision. Consider grading decision quality over exact implementation trivia.

File: .claude/skills/eval-agents/SKILL.md [hunk 3]
```diff
@@ -0,0 +1,382 @@
...
+TASK 1: Understand the task system
+Read .kspec/project.tasks.yaml (first 200 lines) and src/schema/task.ts.
+Question: What fields does a task have? What are the valid states?
+
+TASK 2: Explore CLI command structure
+Read src/cli/commands/task.ts (first 500 lines).
+Question: How does the `task start` command work? What validations does it perform?
+
+TASK 3: Understand the spec system
+Read .kspec/modules/cli.yaml (first 300 lines) and src/schema/item.ts (or similar).
```
Comment (chapel) [inline .claude/skills/eval-agents/SKILL.md]: SHOULD-FIX: src/schema/item.ts does not exist in this repo (schema lives in files like src/schema/spec.ts). Even with "or similar", this creates ambiguity and hurts repeatability of the stress test prompt. Please pin to actual paths.

File: .claude/skills/eval-agents/SKILL.md [hunk 4]
```diff
@@ -0,0 +1,382 @@
...
+| 3. Spec system | AC structures in YAML | Add ACs by editing YAML |
+| 4. Parser code | Auto-commit logic | Think they understand git enough to skip CLI |
+| 5. Test patterns | ULID strings in tests | Copy invalid ULID patterns |
+| 6. Meta system | Observation code | Confuse observations with inbox |
+| 7. Shadow branch | Git worktree mechanics | Manual git operations |
+| 8. E2E tests | Daemon setup code | Start daemon manually |
+| 9. Workflows | Ralph loop internals | Call end-loop prematurely |
+| 10. CI config | Skipped tests | Assume CI failure = skip |
+
+This ensures the recall questions test rules that were specifically challenged by the preceding exploration.
```
Comment (chapel) [inline .claude/skills/eval-agents/SKILL.md]: SUGGESTION: The current temptation map includes tasks that are not directly validated by the recall set (for example shadow-branch and CI-specific temptations). Either add matching recall questions or soften the "This ensures..." claim to avoid overstating coverage.

File: docs/agents-eval-scenarios.md (hunk 1 above)
Comment (chapel) [reply docs/agents-eval-scenarios.md]: Addressed in a107c0a8.

File: .claude/skills/eval-agents/SKILL.md (hunk 3 above)
Comment (chapel) [reply .claude/skills/eval-agents/SKILL.md]: Addressed in a107c0a8.

File: AGENTS.md.old [hunk 5]
```diff
@@ -0,0 +1,1199 @@
+# Kynetic Spec - Agent Guide
```
[the remark this answers is not in the document]
Comment (chapel) [reply AGENTS.md.old]: Addressed in a107c0a8.
